### ISSUE TITLE
Serialisation issue

### ISSUE THREAD

**Issue by y-nk** [ravendb/ravendb-nodejs-client#296]
When storing a value of a model, the SDK crashes under the following code:

```
const nullObj = Object.create(null)

ravenModel.someProp = nullObj
await session.saveChanges()
```

The reason is that nullObj doesn't have a prototype, which is used to determine the type of data (in `Utility/TypeUtil`)

**Comment by ml054** [comment]
https://issues.hibernatingrhinos.com/issue/RDBC-543

@y-nk thanks for report. 

Issue was fixed.

**Comment by ml054** [comment]
Fixed in 5.2.1
